pyscript.core/tests/integration/test_02_display.py

```diff
@@ -457,7 +457,8 @@ def test_image_renders_correctly(self):
                 img = Image.new("RGB", (4, 4), color=(0, 0, 0))
                 display(img, target='img-target', append=False)
             </script>
-            """
+            """,
+            timeout = 30 * 1000,
```

Comment:
this is the same we have now, is this a leftover?

Reply:
good catch, I removed it